ethereum/referencetests/src/main/java/org/hyperledger/besu/ethereum/referencetests/ReferenceTestProtocolSchedules.java

```diff
@@ -106,14 +106,7 @@ public static ReferenceTestProtocolSchedules create(final StubGenesisConfigOptio
                 Map.entry(
                     "CancunToPragueAtTime15k",
                     createSchedule(genesisStub.clone().cancunTime(0).pragueTime(15000))),
-                Map.entry(
-                    "Prague",
-                    createSchedule(
-                        genesisStub
-                            .clone()
-                            .pragueTime(0)
-                            .osakaTime(0) // TODO remove this once osaka_devnet_0 ships
```

Comment:
@shemnon any issue with removing this line?

Removing osakaTime from Prague Schedule fixes some issues when using [ethereum/execution-spec-tests@pectra-devnet-5%40v1.2.0 (release)](https://github.com/ethereum/execution-spec-tests/releases/tag/pectra-devnet-5%40v1.2.0).

Reply:
This was a there when EOF tests were still in the strange prague fork.  OK to remove.